themes/bootstrap3/templates/combined/results-list.phtml

```diff
@@ -74,6 +74,7 @@
       'params' => $params,
       'showCheckboxes' => $this->showCartControls || $this->showBulkOptions,
       'excludeResultIds' => true, // Don't add id attributes to results to avoid duplicates
+      'combined' => true,
```

Comment:
Maybe add a comment explaining why this is here, since it's not used in core code -- e.g. `// Flag that we're in combined mode; may be useful to customize template behavior`

Reply:
Good call.  Done.